plc4j/utils/opm/src/test/resources/logback.xml

```diff
@@ -29,7 +29,7 @@
     </encoder>
   </appender>
 
-  <root level="debug">
+  <root level="trace">
```

Comment:
might be intentional or an accident but trace is a bit much for the poor buildserver ;)